_docs/best-practices.md

```diff
@@ -185,6 +195,8 @@ s:dateCreated: "2016-12-13"
 s:license: https://www.apache.org/licenses/LICENSE-2.0
 ```
 
+### WDL
+
 For WDL descriptors, see the [WDL documentation](https://software.broadinstitute.org/wdl/documentation/spec#metadata-section) for how to define metadata.  
 
 Additionally, this following example includes author, email, and description metadata:
```

Comment:
Can't comment on the image directly, but please generate a more professional example for this kind of user facing documentation

Reply:
not sure which image you're referring to